tools/sz_repo_cli/lib/src/commands/src/build_android_command.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:sz_repo_cli/src/common/common.dart';
+
+final _androidStages = [
+  'stable',
+  'alpha',
+];
+
+/// The different flavors of the Android app.
+final _androidFlavors = [
+  'prod',
+  'dev',
+];
+
+/// The different output types of the Android app.
+final _androidOutputType = [
+  'appbundle',
+  'apk',
+];
+
+class BuildAndroidCommand extends Command {
+  final SharezoneRepo _repo;
+
+  BuildAndroidCommand(this._repo) {
+    argParser
+      ..addOption(
+        releaseStageOptionName,
+        abbr: 's',
+        allowed: _androidStages,
+        defaultsTo: 'stable',
+      )
+      ..addOption(
+        outputTypeName,
+        help: 'The type of output type, either "appbundle" or "apk".',
+        allowed: _androidOutputType,
+        defaultsTo: 'appbundle',
+      )
+      ..addOption(
+        buildNumberOptionName,
+        help: '''An identifier used as an internal version number.
+Each build must have a unique identifier to differentiate it from previous builds.
+It is used to determine whether one build is more recent than
+another, with higher numbers indicating more recent build.
+When none is specified, the value from pubspec.yaml is used.''',
+      )
+      ..addOption(
+        flavorOptionName,
+        allowed: _androidFlavors,
+        help:
+            'The flavor to build for. At the moment only "prod" is supported.',
+        defaultsTo: 'prod',
+      );
+  }
+
+  static const releaseStageOptionName = 'stage';
+  static const flavorOptionName = 'flavor';
+  static const buildNumberOptionName = 'build-number';
+  static const outputTypeName = 'output-type';
+
+  @override
+  String get description =>
+      'Build the Sharezone Android app in release mode. Codemagic CLI tools must be installed.';
+
+  @override
+  String get name => 'android';
+
+  @override
+  Future<void> run() async {
+    // Its less work to just print everything right now instead of selectively
+    // print and add custom print statements for non-verboes output.
+    // One might add non-verbose output in the future but right now this is
+    // easier.
```

Comment:
```suggestion
    // Is used so that runProcess commands print the command that was run.
    // Right now this can't be done via an argument. 
    // This workaround should be addressed in the future.
```

I think this makes it clearer (I know its still the same in the other place but at least its better for this PR right now)